app/common/types/types.py

```diff
@@ -43,3 +41,28 @@ def __modify_schema__(cls, field_schema: Dict[str, str]):
                 " long"
             ),
         )
+
+
+class UsernameStr(str):
+    @classmethod
+    def __get_validators__(cls):
+        yield str_validator
+        yield cls.validate
+
+    @classmethod
+    def __modify_schema__(cls, field_schema: Dict[str, str]):
+        field_schema.update(
+            examples="1234u",
+            description=(
+                "UsernameStr has contain at least one letter                  "
```

Comment:
Looks like too many gaps here, can u check how it looks in swagger? 0.0.0.0/docs

Reply:
<img width="921" alt="image" src="https://user-images.githubusercontent.com/76059151/196180306-297a1bf4-afe9-416f-8c84-949c2f03796e.png">
It looks pretty cool, except one extra backspace

Reply:
Interesting that it looks nice in swagger. But it will be better to delete the extra spaces in code.
Like below
```suggestion
                "UsernameStr has contain at least one letter"
```